src/components/content-header-card/content-header-card.module.css

```diff
@@ -0,0 +1,161 @@
+.contentHeaderCard {
+	padding: 0;
+
+	& .cardTop {
+		padding: 16px;
+		display: flex;
+
+		& .icon {
+			margin-right: 12px;
+			display: none;
+
+			@media (--dev-dot-tablet-up) {
+				display: block;
+			}
+		}
+
+		& .content {
+			flex-grow: 1;
+
+			& .upper {
+				display: flex;
+				gap: 16px;
+				flex-direction: column-reverse;
+				@media (--dev-dot-tablet-up) {
+					flex-direction: row;
+				}
+
+				& .left {
+					flex-grow: 1;
+
+					& .titleWrap {
+						display: flex;
+						flex-wrap: wrap;
+						align-items: center;
+						@media (--dev-dot-tablet-up) {
+							flex-wrap: unset;
+						}
+
+						& .title {
+							color: var(--token-color-foreground-strong);
+						}
+
+						& .attribution {
+							color: var(--token-color-foreground-faint);
+							margin-left: 4px;
+						}
+					}
+
+					& .description {
+						margin-top: 2px;
+						color: var(--token-color-foreground-primary);
+					}
+				}
+
+				& .dropdown {
+					flex-shrink: 0;
+					width: 100%;
+					& > button {
+						width: 100%;
+					}
+
+					@media (--dev-dot-tablet-up) {
+						width: auto;
+						& > button {
+							width: auto;
+						}
+					}
+				}
+			}
+		}
+
+		& .lower {
+			display: flex;
+			flex-direction: column;
+			align-items: flex-start;
+			@media (--dev-dot-tablet-up) {
+				flex-direction: row;
+				align-items: flex-end;
+			}
+
+			& .badgeList {
+				margin-top: 16px;
+			}
+
+			& .note {
+				color: var(--token-color-foreground-faint);
+				flex-grow: 1;
+				text-align: right;
+				margin-top: 12px;
+				@media (--dev-dot-tablet-up) {
+					margin-top: 6px;
+				}
+			}
+		}
+	}
+
+	& .cardBottom {
+		padding: 12px 16px;
+		background: var(--token-color-surface-faint);
+
+		/* This is the color from --token-surface-base-box-shadow,
+		which the color value from it is not in a variable alone */
+		border-top: 1px solid rgba(101, 106, 118, 0.2);
```

Comment:
I'd prefer to not deviate from the system here unless we have to. What's the context on not just using box-shadow here?

Reply:
This is a box in a box (which already has box shadow), because it is a transparent box shadow the left, bottom, and right are .2 transparency + .2 transparency when using box shadow.

Reply:
To visualize their positioning.  Red overlaps orange.


![CleanShot 2023-02-28 at 16 29 17@2x](https://user-images.githubusercontent.com/2105067/222014225-8d3e8740-7464-4bed-8d4b-dae79c9e8ea6.png)

Reply:
ahhah, thank you! This is much more preferable than what I have.